text/0027-filter-and-facet-behavior.md

```diff
@@ -179,20 +180,26 @@ Here is the summary of all the breaking changes (that are detailed in the paragr
 - The `filters` parameter can accept both syntax: string (with `OR`/`AND`) and array.
 - The `:` operator does not exist anymore (was previously present in `facetFilters` in v0.20.0) and is replaced by `=`.
 - The `facetsDistribution` is now applied after the `filters` parameter. This point is currently not documented, not sure this is useful to add it to the docs.
-- All the integer in the user documents are converted into float. So we integers with high values lose precision. However, integers from −2^53 to 2^53 (−9007199254740992 to 9007199254740992) can be exactly represented, which is enough in 99% of cases. Not sure this is important to documented it either.
+- All the integer in the user documents are converted into float. So integers with high values lose precision. However, integers from −2^53 to 2^53 (−9007199254740992 to 9007199254740992) can be exactly represented, which is enough in 99% of cases. Not sure this is important to documented it either.
 
 ### Impact on Documentation
 
-// TODO
+See the previous part.
 
 ## 2. Technical Specifications
 
-// TODO
-
 ### Architecture
+
+Internal MeiliSearch engine uses an index data structure for each type encountered in an attribute. Doing this eliminate the need for the user to strictly type the attribute and permits to deal with loosely typed document at indexation. Thus, facilitating the developper experience.
```

Comment:
@Kerollmops I'm not quite 100% sure about this part. Don't hesitate to point out any errors. Thanks!